UI/ChatUI/ChatUI/ChatUI/Behaviors/ReversedPointerWheelBehavior.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml;
+using Windows.System;
+using Microsoft.UI.Xaml.Input;
+
+namespace ChatUI.Behaviors;
+
+public static class ReversedPointerWheelBehavior
+{
+	public static readonly DependencyProperty IsReversedProperty = DependencyProperty.RegisterAttached(
```

Comment:
IMHO Either we have `PointerWheelBehavior` with `IsReversed` property, either we have `ReversedPointerWheelBehavior` with `IsEnabled` property, but we should not have a double "reversed".

Also personally I'm against the suffix `Behavior` for stuff like here where there is no need ... but we should follow the standard of what was done in the toolkit.

Reply:
I have renamed it to `PointerWheel`